### ISSUE TITLE
Error in recognizeUsingWebSocket by passing MicrophoneInputStream(true)

### ISSUE THREAD

**Issue by ShreeyaJoshi** [watson-developer-cloud/android-sdk#54]
Hi, 
I tried creating Cordova plug-in by following example and sample code given in documentation. 
```java
speechService.recognizeUsingWebSocket(new MicrophoneInputStream(true),
                        getRecognizeOptions(), new BaseRecognizeCallback() {...}
```

When tried passing **OPUS encoded: true** for input stream getting errors,
Stacktrace for same as below:
```

07-11 18:32:39.822 25651-25737/com.example.hello E/AndroidRuntime: FATAL EXCEPTION: Thread-6
    Process: com.example.hello, PID: 25651
    java.lang.NoClassDefFoundError: Failed resolution of: Lcom/sun/jna/Library;
        at com.ibm.watson.developer_cloud.android.library.audio.MicrophoneCaptureThread.run(MicrophoneCaptureThread.java:68)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "com.sun.jna.Library" on path: DexPathList[[zip file "/data/app/com.example.hello-1/base.apk", zip file "/data/app/com.example.hello-1/split_lib_dependencies_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_0_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_1_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_2_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_3_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_4_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_5_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_6_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_7_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_8_apk.apk", zip file "/data/app/com.example.hello-1/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/com.example.hello-1/lib/arm, /data/app/com.example.hello-1/base.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_dependencies_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_0_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_1_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_2_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_3_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_4_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_5_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_6_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_7_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_8_apk.apk!/lib/armeabi-v7a, /data/app/com.example.hello-1/split_lib_slice_9_apk.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at com.ibm.watson.developer_cloud.android.library.audio.MicrophoneCaptureThread.run(MicrophoneCaptureThread.java:68) 
```    

Can you please let me know if I am missing anything here?

**Comment by germanattanasio** [comment]
Please provide a set of reproducible steps.

**Comment by germanattanasio** [comment]
And please use the latest version. I think this could have been fixed in #63

**Comment by lpatino10** [comment]
Closing due to inactivity.
